xdem/spstats.py

```diff
@@ -91,6 +90,73 @@ def random_subset(dh: np.ndarray, coords: np.ndarray, nsamp: int):
 
     return dh_sub, coords_sub
 
+def create_circular_mask(h: float, w: float, center: Optional[list[float]] = None, radius: Optional[float] = None) -> np.ndarray:
+    """
+    Create circular mask on a raster
+
+    :param h: height of array
+    :param w: width of array
+    :param center: center
```

Comment:
Good to mention what the default will be, as it's unclear from the function definition.

Reply:
Done